### PR TITLE
Update configure-dynamic-emergency-calling.md

### PR DESCRIPTION
**Pull request by kellybos**
Working with customer and engineering confirmed that Teams phones will not support WAP - https://portal.microsofticm.com/imp/v3/incidents/details/231579196/home.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/configure-dynamic-emergency-calling.md [hunk 1]
```diff
@@ -81,6 +81,10 @@ The following clients are currently supported.  Check back often to see updates
 
 > [!NOTE]
 > Dynamic emergency calling including security desk notification isn't supported on the Teams web client. To prevent users from using the Teams web client to call PSTN numbers, you can set a Teams calling policy and turn off the **Allow web PSTN calling** setting. To learn more, see [Calling policies in Teams](teams-calling-policy.md) and [Set-CsTeamsCallingPolicy](https://docs.microsoft.com/powershell/module/skype/set-csteamscallingpolicy?view=skype-ps).
+> 
+
+> [!NOTE]
+> Dynamic emergency calling on Teams phones will not suport WAP for LIS matching.
```
Comment (JohanFreelancer9) [inline Teams/configure-dynamic-emergency-calling.md:87]:
```suggestion
> Dynamic emergency calling on Teams phones does not support WAP for LIS matching.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@kellybos - please commit my suggestion.

**Comment by CarolynRowe** [comment]
@scanum This PR branch has unresolved conflicts.  Can someone please resolve before I  merge?  Thank you.

**Comment by kellybos** [comment]
When will this update to the documentation be released? This is critical limitation for customers planning deployments to be aware of.

**Comment by JohanFreelancer9** [comment]
@kellybos - please commit my suggestion. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by CarolynRowe** [comment]
@JohanFreelancer9 I am checking with the PM on this request.  Please don't merge until we get the OK.

**Comment by CarolynRowe** [comment]
Recent discussion with the PM confirmed that this is supported.  If this is incorrect, please contact technical support or resubmit another PR.  Thank you.

**Pull request closed without merging**